yt_dlp/extractor/veo.py

```diff
@@ -0,0 +1,74 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+from ..utils import (
+    int_or_none,
+    mimetype2ext,
+    unified_timestamp,
+    url_or_none,
+)
+
+
+class VeoIE(InfoExtractor):
+    _VALID_URL = r'https?://app\.veo\.co/matches/(?P<id>[0-9A-Za-z-]+)'
+
+    _TEST = {
+        'url': 'https://app.veo.co/matches/20201027-last-period/',
+        'info_dict': {
+            'id': '20201027-last-period',
+            'ext': 'mp4',
+            'title': 'Akidemy u11s v Bradford Boys u11s (Game 3)',
+            'thumbnail': 'https://c.veocdn.com/8733223a-c6a0-4947-a309-f0f9fb44a1e4/standard/machine/8011e457/thumbnail.jpg',
```

Comment:
```suggestion
            'thumbnail': 're:https://c.veocdn.com/.+/thumbnail.jpg',
```
no need to hardcode the url